metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -607,9 +594,22 @@ def detect_quickstart_arch(arch: Optional[str]) -> Architectures:
     help="Specify the architecture for the quickstart images to use. Options are x86, arm64, m1 etc.",
 )
 @upgrade.check_upgrade
-@telemetry.with_telemetry
+@telemetry.with_telemetry(
+    kwargs=[
```

Comment:
Curious - what is this doing?

Reply:
this tells telemetry to capture that list of kwargs from the function call

Reply:
updated to make it more clear